src/assets/js/merkle_graph_d3.js

```diff
@@ -45,22 +46,46 @@ export function load_merkle_graph_js(tree_data){
 
     svg.call(d3.zoom().on('zoom', (e) => {
         zoom_g.attr('transform', e.transform)
-    }))    
-
-    svg_g.selectAll('path')
-        .data(links)
-        .enter()
-        .append('path')
-            .attr('d', d => {
-                let halfway_y = (d.target.y + d.source.y) / 2
-                return horizontal
-                    ? `M${d.source.y} ${d.source.x} ${halfway_y} ${d.source.x} ${halfway_y} ${d.target.x} ${d.target.y} ${d.target.x}`
-                    : `M${d.source.x} ${d.source.y} ${d.source.x} ${halfway_y} ${d.target.x} ${halfway_y}  ${d.target.x} ${d.target.y}`
-            })
-
-    let node_groups = svg_g.selectAll('g')
+    }))
+
+    let line_groups = svg_g.selectAll('g.lines')
+    .data(links)
+    .join("g")
+    .attr('class', 'lines')
+
+    let arrow_length = 10
+    let arrow_width = 10
+    // shorten line by this amount to keep it from showing behind arrow point
```

Comment:
1e2fa309f0a0b11ffdcffad18fb5b9573b02ee58: Arrow tips might be more confusing than helpful. There are multiple relationships between nodes in the tree and the arrow pointing from the child to the parent is not always appropriate. I will discuss this with the others from the research team.

Reply:
Ok, please remove the arrow tips.